contrib/grpc_metrics/src/test/java/io/opencensus/contrib/grpc/metrics/RpcViewsTest.java

```diff
@@ -37,93 +39,57 @@
   public void registerCumulative() {
     FakeViewManager fakeViewManager = new FakeViewManager();
     RpcViews.registerAllCumulativeViews(fakeViewManager);
-    verifyRegistration(
-        fakeViewManager,
-        /* isOldRpcCumulativeViewsRegistered= */ true,
-        /* isOldRpcIntervalViewsRegistered= */ false,
-        /* isNewRpcClientViewsRegistered= */ false,
-        /* isNewRpcServerViewsRegistered= */ false);
+    assertThat(fakeViewManager.getRegisteredViews())
+        .containsExactlyElementsIn(RpcViews.RPC_CUMULATIVE_VIEWS_SET);
   }
 
   @Test
   public void registerInterval() {
     FakeViewManager fakeViewManager = new FakeViewManager();
     RpcViews.registerAllIntervalViews(fakeViewManager);
-    verifyRegistration(
-        fakeViewManager,
-        /* isOldRpcCumulativeViewsRegistered= */ false,
-        /* isOldRpcIntervalViewsRegistered= */ true,
-        /* isNewRpcClientViewsRegistered= */ false,
-        /* isNewRpcServerViewsRegistered= */ false);
+    assertThat(fakeViewManager.getRegisteredViews())
+        .containsExactlyElementsIn(RpcViews.RPC_INTERVAL_VIEWS_SET);
+  }
+
+  @Test
+  public void registerAll() {
+    FakeViewManager fakeViewManager = new FakeViewManager();
+    RpcViews.registerAllViews(fakeViewManager);
+    assertThat(fakeViewManager.getRegisteredViews())
+        .containsExactlyElementsIn(
+            ImmutableSet.builder()
+                .addAll(RpcViews.RPC_CUMULATIVE_VIEWS_SET)
+                .addAll(RpcViews.RPC_INTERVAL_VIEWS_SET)
+                .build());
   }
 
   @Test
   public void registerAllGrpcViews() {
     FakeViewManager fakeViewManager = new FakeViewManager();
     RpcViews.registerClientGrpcViews(fakeViewManager);
     RpcViews.registerServerGrpcViews(fakeViewManager);
```

Comment:
It looks like this test is no longer testing RpcViews.registerAllGrpcViews().

Reply:
Good catch, I added a similar helper method to register both client and server views.